src/clients/SnapshotGraphqlTypes.ts

```diff
@@ -21,7 +21,7 @@ export type SnapshotSpace = {
 }
 export type SnapshotVoteResponse = SnapshotQueryResponse<{ votes: SnapshotVote[] }>
 export type SnapshotVote = {
-  id?: string
+  id: string
```

Comment:
This should remain optional unless we make sure that every query sent to graphql returns the vote `id`